javatests/arcs/android/integration/policy/scenarios.arcs

```diff
@@ -38,3 +43,31 @@ recipe PersistsEgresses
   // And the recipe egresses Thing {a, b}
   EgressAB
     output: handle1
+
+/// Scenario: A policy-compliant recipe with @ttl handles egresses data and
+/// stores ingress-restricted values to RAM that are deleted when the TTL expires
+/// and at the runtime’s end.
+
+@egressType('GeneralEgress')
+policy SimpleTtlPolicy {
+  @allowedRetention(medium: 'Ram', encryption: false)
+  from Thing access {
+    a,
+    b
+  }
+}
+
+// Given a recipe is compliant with a policy
+@policy('SimplePersistencePolicy')
+recipe TtlEgresses
+  // And the recipe's store is marked @ttl('2h')
+  handle1: create @ttl('2h')
```

Comment:
Create another handle with `1h` ttl? That way you can run a test to like this: 
  - Write data to two handles.
  - trigger clean up after an hour: data in 1h ttl is gone, but data with 2h ttl is still present
  - trigger clean up after another hour: all data is gone.

Reply:
I like it.